Automattic/wp-calypso
# Add auto-renew toggle on the new registered domain screen

#### Changes proposed in this Pull Request

* Display the auto-renew toggle on the registered domain screen
* Display "expiring credit card" message for cases where the credit card expires/expired before renewal time

Here's how it looks with credit card that is expiring before the domain renewal:

<img width="742" alt="Screenshot 2020-03-18 at 0 09 40" src="https://user-images.githubusercontent.com/1355045/76906738-4e450b00-68ad-11ea-9e43-c922163dfe3e.png">

![Screenshot 2020-03-18 at 0 10 29](https://user-images.githubusercontent.com/1355045/76906803-69b01600-68ad-11ea-9ec7-65794af7b2f9.png)

<img width="553" alt="Screenshot 2020-03-18 at 0 10 35" src="https://user-images.githubusercontent.com/1355045/76906765-59983680-68ad-11ea-8584-dcb3ee7b2783.png">

<img width="755" alt="Screenshot 2020-03-18 at 0 10 43" src="https://user-images.githubusercontent.com/1355045/76906782-6157db00-68ad-11ea-8347-ab4ca3c5499d.png">

#### Testing instructions

* Open a registered/mapped domain paid with a credit card and verify that you can enable/disable the auto-renew toggle
* Verify that the expiring credit card notification is shown
* Verify that the message is correct in the expiring soon notification for mapped domains/registered domains
* Verify that domain purchased with credits looks okay - there should be no auto-renew toggle

Also fixes #40067
